subsys/bluetooth/host/l2cap.c

```diff
@@ -853,14 +853,27 @@ static bool chan_has_credits(struct bt_l2cap_le_chan *lechan)
 #endif
 }
 
+__weak void bt_l2cap_data_pull_spy(struct bt_conn *conn,
+				   struct bt_l2cap_le_chan *lechan,
+				   size_t amount,
+				   size_t *length)
+{
+}
```

Comment:
Suggest to incorperate `test` in the name of this function if it's specifically only for testing.